sdk/servicebus/azure-servicebus/samples/sync_samples/mgmt_rule.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+Example to show managing rule entities under a ServiceBus Namespace, including
```

Comment:
Probably worth a mention that this is for subscription/topic and that queue level rules are comparable but with queue specific calls.